core/src/main/scala/org/broadinstitute/dsde/rawls/metrics/RawlsInstrumented.scala

```diff
@@ -50,19 +45,6 @@ trait RawlsInstrumented extends WorkbenchInstrumented {
     status => builder
       .expand(WorkflowStatusMetricKey, status)
       .asCounter("count")
-
-  protected def httpRequestMetric(builder: ExpandedMetricBuilder): (HttpRequest, HttpResponse) => ExpandedMetricBuilder = {
```

Comment:
I thought the HTTP metrics stuff should live in the base class `WorkbenchInstrumented` instead of `RawlsInstrumented`. `rawls/metrics` already has a spray dependency and was defining the spray expansions. Other systems (such as Orch) might want to make use of the spray stuff too.